Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -144,39 +146,40 @@ class WebViewController: UIViewController, WebController {
             self.reloadBlockers(lists)
         }
     }
-
-    private func setupUserScripts() {
-        browserView.configuration.userContentController.add(self, name: "focusTrackingProtection")
-        let source = try! String(contentsOf: Bundle.main.url(forResource: "preload", withExtension: "js")!)
-        let script = WKUserScript(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: true)
+    
+    private func addScript(forResource resource: String, injectionTime: WKUserScriptInjectionTime, forMainFrameOnly mainFrameOnly: Bool) {
+        let source = try! String(contentsOf: Bundle.main.url(forResource: resource, withExtension: "js")!)
+        let script = WKUserScript(source: source, injectionTime: injectionTime, forMainFrameOnly: mainFrameOnly)
         browserView.configuration.userContentController.addUserScript(script)
+    }
 
+    private func setupTrackingProtectionScripts() {
+        browserView.configuration.userContentController.add(self, name: "focusTrackingProtection")
+        addScript(forResource: "preload", injectionTime: .atDocumentStart, forMainFrameOnly: true)
         browserView.configuration.userContentController.add(self, name: "focusTrackingProtectionPostLoad")
-        let source2 = try! String(contentsOf: Bundle.main.url(forResource: "postload", withExtension: "js")!)
-        let script2 = WKUserScript(source: source2, injectionTime: .atDocumentEnd, forMainFrameOnly: false)
-        browserView.configuration.userContentController.addUserScript(script2)
-        
+        addScript(forResource: "postload", injectionTime: .atDocumentEnd, forMainFrameOnly: false)
+    }
+    
+    private func setupFindInPageScripts() {
         browserView.configuration.userContentController.add(self, name: "findInPageHandler")
-        let source3 = try! String(contentsOf: Bundle.main.url(forResource: "FindInPage", withExtension: "js")!)
-        let script3 = WKUserScript(source: source3, injectionTime: .atDocumentEnd, forMainFrameOnly: true)
-        browserView.configuration.userContentController.addUserScript(script3)
+        addScript(forResource: "FindInPage", injectionTime: .atDocumentEnd, forMainFrameOnly: true)
     }
 
     func disableTrackingProtection() {
```

Comment:
The crash was occurring when enabling tracking protection because the FindInPageHandler was being added again (and never removed). To fix this, we now remove all the handlers in disable and add the FindInPageHandler in disable as well.

Reply:
Instead of disabling them all and re-adding find in page. Could you just toggle the tracking protection scripts?

Reply:
What do you mean by toggle? It doesn't look like we can individually remove scripts (we tried iterating through them and only removing if it's not find in page, but you can only run a removeAll).

Reply:
Ahh, why would you want to do a silly thing like that! I guess I could have looked that up! Sorry!